src/pages/NotificationsPage/components/notification-dropdown.tsx

```
@@ -0,0 +1,285 @@
+                <div className="relative" ref={filterRef}>
+                  <button
+                    type="button"
+                    onClick={() => setShowFilter((prev) => !prev)}
+                    className="flex items-center gap-1 rounded px-2 py-1 text-xs font-medium text-gray-500 hover:bg-gray-100"
+                  >
+                    {filter === "all" ? "All" : "Unread"}
+                    <svg
+                      className="h-3.5 w-3.5"
+                      viewBox="0 0 20 20"
+                      fill="currentColor"
+                    >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)